profile/README.md

```diff
@@ -1,13 +1,9 @@
-<img src="https://github.com/w3f/Grants-Program/blob/master/src/Grants_Program.png" style="width:1300px; display: block; margin-left: auto; margin-right: auto;">
```

Comment:
What's wrong with a header image?

Reply:
Well, this is not the grants program, it's just the archive. I don't think the banner belongs everywhere related to the grants program but should just be used as a landing banner in the README where it is right now. This place is not a huge thing and doesn't need to look like one.

Reply:
I'll go ahead and merge this. Maybe it doesn't look better without it, we'll see.

Reply:
I'll go ahead and merge this. Maybe it doesn't look better without it, we'll see.